### PR TITLE
1142 remove family member

### PR DESCRIPTION
**Pull request by aurangzaibumer**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes [#1142](https://github.com/opensrp/fhircore/issues/1142) 

**Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [x] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the fhircore app to verify my change fixes the issue and/or does not break the app

### CODE REVIEW

File: android/anc/src/main/java/org/smartregister/fhircore/anc/ui/details/PatientDetailsActivity.kt [hunk 1]
```diff
@@ -201,14 +201,24 @@ class PatientDetailsActivity : BaseMultiLanguageActivity() {
...
+            )
         )
+//        AlertDialogue.showConfirmAlert(
+//          this,
+//          R.string.remove_this_person_confirm_message,
+//          R.string.remove_this_person_confirm_title,
+//          this::onDeleteFamilyMemberRequested,
+//          R.string.remove_this_person_button_title,
+//          DeletionReason.values().map { AlertDialogListItem(it.name, getString(it.label)) }
+//        )
```
Comment (dubdabasoduba) [inline android/anc/src/main/java/org/smartregister/fhircore/anc/ui/details/PatientDetailsActivity.kt:221]: Remove this dead code if you are no longer using it.

File: android/anc/src/main/java/org/smartregister/fhircore/anc/ui/family/form/RemoveFamilyQuestionnaireActivity.kt [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        //remove the family member data from resourceEntity
+        questionnaireViewModel.process(
+        intent.getStringExtra(QUESTIONNAIRE_ARG_PATIENT_KEY),
+        questionnaire,
+        questionnaireResponse
+        )
+
+    }
+
+
```
Comment (dubdabasoduba) [inline android/anc/src/main/java/org/smartregister/fhircore/anc/ui/family/form/RemoveFamilyQuestionnaireActivity.kt:107]: Reformat this to remove the extra line.

File: android/anc/src/main/java/org/smartregister/fhircore/anc/ui/family/form/RemoveFamilyQuestionnaireActivity.kt [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+    override val questionnaireViewModel by viewModels<RemoveFamilyQuestionnaireViewModel>()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setupUI()
+        loadData()
+    }
+
+    private fun loadData() {
+        familyId = intent.extras?.getString(QUESTIONNAIRE_ARG_PATIENT_KEY) ?: ""
```
Comment (dubdabasoduba) [inline android/anc/src/main/java/org/smartregister/fhircore/anc/ui/family/form/RemoveFamilyQuestionnaireActivity.kt:35]: @ellykits Same question I have earlier, can we create an object for the family & patient. We can then pass this object to the other classes any time we need to.  Right now we are passing single attributes (which is okay) but will it be too many intent etxras at some point?

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]

**Review by dubdabasoduba — DISMISSED** [review]

**Review by dubdabasoduba — APPROVED** [review]

**Review by dubdabasoduba — DISMISSED** [review]

**Comment by ellykits** [comment]
Closing this. Implementation already merged to main via another PR.

**Pull request closed without merging**